src/components/button/button.js

```diff
@@ -14,14 +14,21 @@ export default function Button({
   startEnhancer,
   endEnhancer,
   overrides = {},
+  $size,
+  $kind,
```

Comment:
We only use the $ prefix on styled components. For the main exports you should just use normal props `size` and `kind`. When they are passed to "BaseButton", "ButtonLabel", etc, that's when we prefix with a dollar.

Reply:
Gotcha, updating to address this.